hack/deploy-kubevirt.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 set -exo pipefail
 
+if [ -z "${OCI_BIN}" ];then
```

Comment:
this is weird, we concluded at the past that we are doing it once on the bootstrap / locally and not here, 
if it is not mandatory please remove it

Reply:
we can't add the -kv, because it installs ipam and we dont want to reinstall it
and also want to control what kubevirt version we install
i would say please add it to the git actions once instead, no need each time
this what also was decided fwiw when we had same discussions on other repos

Reply:
Moved it to the automation script so it only involves ipam-ext lane.